opc-ua-stack/stack-core/src/main/java/org/eclipse/milo/opcua/stack/core/types/builtin/DateTime.java

```diff
@@ -29,7 +30,7 @@
     private final long utcTime;
 
     public DateTime() {
-        this(javaToUtc(System.currentTimeMillis()));
+        this(Instant.now());
```

Comment:
I wonder if the original default constructor is faster. `Instant.now()` seems to do a lot.

Reply:
Pros for Instant: it reserves the nano seconds, while old constructor uses only millis.

Cons: this might be a change in expectations.

No issue, I will revert it.

Reply:
What if we revert the constructor but provide another static function `DateTime.instant()` that gives you easy access to nano-precision DateTime?

Reply:
Making the constructor private would need to happen on `dev/0.3` branch. I'm okay breaking APIs since the library is version `0.x` but I try to keep there breaks to the intermediate version numbers.